doc/z-chap03.xml

```diff
@@ -196,16 +196,17 @@
 
   <Section Label = "creating-bipartitions">
     <Heading>Creating bipartitions</Heading>
-  There are several ways of creating bipartitions in &GAP;, which are
-  described in this section. The maximum degree of a bipartition is set as
-  2 ^ 29 - 1. In reality, it is unlikely to be possible to create bipartitions
-  of degrees as small as 2 ^ 24 because they require too much memory.
+    There are several ways of creating bipartitions in &GAP;, which are
+    described in this section. The maximum degree of a bipartition is set as
+    <M>2 ^ 29 - 1</M>. In reality, it is unlikely to be possible to create bipartitions
+    of degrees as small as <M>2 ^ 24</M> because they require too much memory.
```

Comment:
```suggestion
    <M>2 ^ {29} - 1</M>. In reality, it is unlikely to be possible to create bipartitions
    of degrees as small as <M>2 ^ {24}</M> because they require too much memory.
```